Assets/Scripts/SS3D/Systems/Screens/GameScreensController.cs

```diff
@@ -13,6 +15,21 @@ public sealed class GameScreensController : NetworkActor
         [SerializeField] private bool _blockNone;
         [SerializeField] private bool _menuOpen;
         private PlayerSpawnedState _spawnedState;
+        private Controls.OtherActions _controls;
+
+        private void OnEnable()
+        {
+            try
```

Comment:
replace with` if(null && !didstart) return; else { Debug.Error(); return; }` So we can still have an error in case performed is null later in the game.